main.py

```diff
@@ -1,7 +1,45 @@
 import init_django_orm  # noqa: F401
 
 from django.db.models import QuerySet
+from db.models import Genre, Actor
 
 
 def main() -> QuerySet:
-    pass
+    genres = ["Western", "Action", "Dramma"]
+
+    actors = [
+        ("George", "Klooney"),
+        ("Kianu", "Reaves"),
+        ("Scarlett", "Keegan"),
+        ("Will", "Smith"),
+        ("Jaden", "Smith"),
+        ("Scarlett", "Johansson")
+    ]
+
+    # CREATE
+    for genre in genres:
+        Genre.objects.create(name=genre)
+
+    for f_name, l_name in actors:
+        Actor.objects.create(
+            first_name=f_name,
+            last_name=l_name
+        )
+
+    # UPDATE
+    Genre.objects.filter(name="Dramma").update(name="Drama")
+
+    Actor.objects.filter(
+        first_name="George", last_name="Klooney"
+    ).update(last_name="Clooney")
+
+    Actor.objects.filter(
+        first_name="Kianu", last_name="Reaves"
+    ).update(first_name="Keanu", last_name="Reeves")
+
+    # DELETE
+    Genre.objects.filter(name="Action").delete()
+    Actor.objects.filter(first_name="Scarlett").delete()
+
+    # RETURN
```

Comment:
this comment doesn't provide any additional info, so it is redundant. Fix everywhere